### PR TITLE
Add requireExcoLogIn 

### PR DESCRIPTION
**Pull request by sweiyang**
shifted requirelogin and requiresignup before the catch wrapper as well

### CODE REVIEW

File: src/test/java/seedu/club/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
     public static final String VALID_EMAIL_AMY = "amy@example.com";
     public static final String VALID_EMAIL_BOB = "bob@example.com";
     public static final String VALID_GROUP_AMY = "Exco";
-    public static final String VALID_GROUP_BOB = "logistics";
+    public static final String VALID_GROUP_BOB = "logistic";
```
Comment (yash-chowdhary) [inline src/test/java/seedu/club/logic/commands/CommandTestUtil.java]: isn't the group called "logistics" ?

**Review by yash-chowdhary — COMMENTED** [review]
looks good! just one minor change, possibly.

File: src/test/java/seedu/club/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (sweiyang) [reply src/test/java/seedu/club/logic/commands/CommandTestUtil.java]: Oh i’ll go and fix it.

Comment (sweiyang) [reply src/test/java/seedu/club/logic/commands/CommandTestUtil.java]: FIXED

Comment (yash-chowdhary) [reply src/test/java/seedu/club/logic/commands/CommandTestUtil.java]: cool 👍

**Pull request merged**